README.md

```diff
@@ -161,6 +166,13 @@ to administer Fluo:
 If you run some tests and then want a fresh cluster, run the `setup` command again which will
 kill all running processes, clear any data and logs, and restart your cluster.
 
+## Wipe command
+
+The `wipe` command will kill all running processes for your local development cluster and clear
```

Comment:
`uno wipe`

Reply:
done and updated the setup command to match.